pkg/metrics/otel.go

```diff
@@ -184,14 +184,14 @@ processors:
           - action: aggregate_labels
             # label_set is the allowlist of labels to keep after aggregation
             label_set: [status, operation]
-            aggregation_type: sum
+            aggregation_type: max
       - include: declared_resources
         action: update
         new_name: current_declared_resources
         operations:
           - action: aggregate_labels
             label_set: []
-            aggregation_type: sum
+            aggregation_type: max
       - include: kcc_resource_count
         action: update
         operations:
```

Comment:
should this be max too? it's the count for a single repo. i dont think we want to sum it, otherwise it looks like the repo has twice as many objects

Reply:
I can't find where this is recorded in the code... Do we even send this any more?

Reply:
This is a resource group metric: https://gke-internal.googlesource.com/GoogleCloudPlatform/resource-group/+/refs/heads/master/controllers/metrics/record.go#54

Reply:
huh. might be nice to group the metrics by source so this is more obvious in the future. not important for 1.16.1 tho.

Reply:
Looks like this is the count for all the KCC objs in the ResourceGroup. So max seems correct.

Reply:
That said... do we even need to aggregate this? It doesn't have commit or type does it?

Reply:
Ah you are right this is after the metric is recorded and counter has already incremented. Changed.

Reply:
The purpose for this filter is to constrain the labels so that any future labels added will always be excluded from Monarch pipeline.